packages/driver/src/cy/commands/actions/check.ts

```diff
@@ -79,30 +79,29 @@ const checkOrUncheck = (Cypress, cy, type, subject, values: any[] = [], userOpti
       'Elements': $el.length,
     }
 
-    if (options.log) {
-      // figure out the userOptions which actually change the behavior of clicks
-      const deltaOptions = $utils.filterOutOptions(options)
-
-      options._log = Cypress.log({
-        message: deltaOptions,
-        $el,
-        timeout: options.timeout,
-        consoleProps () {
-          return _.extend(consoleProps, {
-            Options: deltaOptions,
-          })
-        },
-      })
-
-      options._log.snapshot('before', { next: 'after' })
+    // figure out the userOptions which actually change the behavior of clicks
+    const deltaOptions = $utils.filterOutOptions(options)
 
-      // warn cmd requires all subjects to have value when args passed to cmd
-      if (!($el.attr('value')) && (values.length > 0)) {
-        $errUtils.throwErrByPath('check_uncheck.element_missing_value_attribute', {
-          onFail: options._log,
-          args: { node, cmd: type },
+    options._log = Cypress.log({
+      hidden: options.log === false,
+      message: deltaOptions,
+      $el,
+      timeout: options.timeout,
+      consoleProps () {
+        return _.extend(consoleProps, {
+          Options: deltaOptions,
         })
-      }
+      },
+    })
+
+    options._log?.snapshot('before', { next: 'after' })
```

Comment:
Can `Cypress.log` return undefined/null from above?

Reply:
Yes when protocol is disabled and `{ log: false }` is true